### ISSUE TITLE
Practical implication of this projects

### ISSUE THREAD

**Issue by dimityrivanov** [kakopappa/sinric#328]
@kakopappa 
Hello author of Sinric i decided to use the library to make a practical use of it using esp8266 mobile application and voice assistant and I am sharing the information with you here https://github.com/dimityrivanov/ESP-ACController

**Comment by kakopappa** [comment]
@dimityrivanov Thanks mate. Looks awsome.

have you tried making the PCB and soldering the components  yet?

**Comment by dimityrivanov** [comment]
@kakopappa Yeah the PCB is tested already, you can open the schematic it will make it more straight forward how the header is sorted out. One bug thought you have to provide power over the usb to program it small mistake but not a big deal :D

**Comment by kakopappa** [comment]
sure
